mobile/lib/shared/providers/asset.provider.dart

```diff
@@ -76,8 +77,12 @@ class AssetNotifier extends StateNotifier<AssetsState> {
       GroupAssetsBy
           .values[_settingsService.getSetting(AppSettingsEnum.groupAssetsBy)],
     );
-    state = await AssetsState.fromAssetList(newAssetList)
-        .withRenderDataStructure(layout);
+
+    // Rendering will not show archived assets by default - use database provider if
+    // you want to access the isArchived assets
+    state = await AssetsState.fromAssetList(
+      newAssetList.whereNot((asset) => asset.isArchived).toList(),
```

Comment:
@matthinc Do you have any suggestions on cases like this when you design this component? Since the ImmichAssetGrid component takes in RenderList and a list of Asset independently, which make it quite complicated to handle cases like this.

Reply:
I think we either need to make several independent assetProviders (with their own renderList) for each different collection.. or we rework this entirely and try to use pagination and watcher from the Isar DB